### PR TITLE
Feature/sim 1496 new neff

### PR DESCRIPTION
**Pull request by rhiannonlynne**
These are the changes for Neff and the seeing default values. 
I also made an update for the bandpass interpolation, mostly to try to better match what Chuck was doing in his ipython notebooks. If the interpolation update is not desireable, I'm happy to remove that change. It doesn't seem to make a great deal of difference.

### CODE REVIEW

**Comment by rhiannonlynne** [comment]
Although .. I think there is some problem with using splev in the way I was doing, when it's far from the bandpass values. So I've made an update there, which fixed one unit test. 
But then I found out that these changes to neff and the seeing values mean that there are small (but significant) changes in the noise wrt simple calculation of SNR vs full calculation. Not sure what's going on. I'll go talk to Scott in person.

**Comment by rhiannonlynne** [comment]
It was a weird result of changing the bandpass interpolation method. 
So, went back from scipy.splrep to np.interp (very simple linear interpolation).

File: python/lsst/sims/photUtils/LSSTdefaults.py [hunk 1]
```diff
@@ -9,7 +9,8 @@ class LSSTdefaults(object):
     """
 
     def __init__(self):
-        self._seeing = {'u': 0.77, 'g':0.73, 'r':0.70, 'i':0.67, 'z':0.65, 'y':0.63}
+        #self._seeing = {'u': 0.77, 'g':0.73, 'r':0.70, 'i':0.67, 'z':0.65, 'y':0.63}
```
Comment (danielsf) [inline python/lsst/sims/photUtils/LSSTdefaults.py]: You should update the docstring on the seeing attribute of LSSTdefaults to reflect the new meaning of "seeing" (also the fact that these values are no longer consistent with arXiv:0805.2366)

**Comment by danielsf** [comment]
Once the docstring for LSSTdefaults.seeing() is updated, this is good to merge (assuming the unit tests do, indeed, all pass, which I think they do)

**Comment by rhiannonlynne** [comment]
I'm just a little confused on whether this is actually what we want to do for all users. For sys_eng use, I think this is exactly what we should do .. but I'm not sure if for other calculations neff should still be calculated the 'old' way. (for example, for calculating m5 for opsim purposes, if we do this more thorough calculation in the future, I'm not sure whether their seeing would actually be reported as seeing or as FWHMeff). 
I'd like to get Zeljko to let me know what was decided as a way forward at the PST meeting, I think.

**Comment by rhiannonlynne** [comment]
Will rebase on new updated master and merge, with the addition of a bit more documentation on FWHMeff and how to convert to FWHMtot (we seem to have now converged on this from the system engineering point of view).

**Pull request closed without merging**